.github/workflows/etcd_integration.yml

```diff
@@ -0,0 +1,52 @@
+name: etcd_integration
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        etcd-version: [ 'v3.4.30', 'v3.5.12' ]
```

Comment:
I would add a link https://endoflife.date/etcd to the comment.
Feel free to ignore.

Reply:
Fixed.